### DISCUSSION TITLE
Andere Länder

### DISCUSSION THREAD

**Discussion by C1710** [marhei/TrainCompany-Data#118] [Q&A]
Bspw. in der Schweiz gibt es ja nicht für alle Bahnhöfe einen RIL100-Code.
Spricht irgendwas dagegen, da ein Format wie `CH:ZUE` zu verwenden (bzw. alternativ die UIC-Identifikation des Haltepunkts)?
(bzw. ist sowas schon erlaubt?)

**Comment by marhei** [comment]
Bitte aktuell erstmal auf Deutschland und Grenznahe Bahnhöfe beschränken.
